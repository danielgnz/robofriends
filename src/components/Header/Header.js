import React from 'react';
import CounterButton from '../CounterButton/CounterButton';

const Header = () => {
	return (
		<header>
			<h1 class="tc f1 sega t-shadow">RoboFriends</h1>
			<CounterButton color="green"/>
		</header>
	);
}

export default Header;