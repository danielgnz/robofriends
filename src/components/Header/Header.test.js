import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

it('expected to render Header component', () => {
	expect(shallow(<Header />)).toMatchSnapshot();
})