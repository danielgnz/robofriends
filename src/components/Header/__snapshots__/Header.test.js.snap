// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expected to render Header component 1`] = `
<header>
  <h1
    class="tc f1 sega t-shadow"
  >
    RoboFriends
  </h1>
  <CounterButton
    color="green"
  />
</header>
`;
